.github/workflows/on-merge.yml

```diff
@@ -12,49 +12,55 @@ jobs:
     name: Build module
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v3
+    - uses: actions/checkout@v4
     - name: Set environment variables from parameters
       shell: bash
       run: |
         echo "NEXUS_USERNAME=${{ secrets.NEXUS_USERNAME }}" >> $GITHUB_ENV
         echo "NEXUS_PASSWORD=${{ secrets.NEXUS_PASSWORD }}" >> $GITHUB_ENV
-    - uses: actions/cache@v1
+    - uses: actions/cache@v4
       with:
         path: ~/.m2/repository
         key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
         restore-keys: |
           ${{ runner.os }}-maven-
     - name: Set up JDK 1.8
-      uses: actions/setup-java@v3
+      uses: actions/setup-java@v4
       with:
         java-version: 8
         distribution: 'temurin'
         cache: 'maven'
     - name: Build with Maven
-      run: mvn -s .github/maven.settings.xml -ntp -B -Papache-release -DskipTests -Dmaven.javadoc.skip=true -Dgpg.skip=true --file pom.xml install
+      run: mvn -s .github/maven.settings.xml -ntp -B -Papache-release -DskipTests -Dmaven.javadoc.skip=true -Dgpg.skip=true -Drat.skip=true --file pom.xml install
 
   publish:
     name: Publish module
-    needs: [build]
+    needs: [ build ]
+    if: github.ref == 'refs/heads/2.20-jahia'
     runs-on: ubuntu-latest
+    container:
+      image: jahia/cimg-mvn-cache:ga_cimg_openjdk_11.0.20-node
```

Comment:
ok to use jdk 11 here but use jdk 1.8 in the maven deploy? or do we switch this to jdk 8?